src/plugins/visualizations/public/embeddable/disabled_lab_visualization.scss

```diff
@@ -0,0 +1,4 @@
+.visDisabledLabVisualization {
+  width: 100%;
+  height: fit-content
```

Comment:
What is the purpose of using `fit-content` here? I'm curious if it's actually doing anything, as it appears to be missing a formula. Also, it would appear that this usage isn't supporting in Firefox currently.